waiter/src/waiter/auth/saml.clj

```diff
@@ -203,8 +203,8 @@
         ; https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims
         upn (first (get attrs "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn"))
         saml-principal (or upn name-id-value)
-        {:keys [authorization/principal authorization/user]} (auth/auth-params-map :saml saml-principal)
```

Comment:
Avoid unnecessarily building the `auth-params-map` just for the principal and user.